### ISSUE TITLE
Composer and standard installation from git releases is a mess

### ISSUE THREAD

**Issue by Z3NTL3** [phpmyadmin/phpmyadmin#19016]
I am using phpmyadmin for my school project to share a related DB through a mysql client to ease our deveopment however;

So basically, I am facing with errors thrown by phpmyadmin meanwhile I have installed all dependencies and requirements all accordingly.
Which is to weird like I never faced that before when trying an open source project.

Reverse Proxy:
![image](https://github.com/phpmyadmin/phpmyadmin/assets/48758770/c7f06353-a486-440b-bd0b-ae503c050951)

Yarn:
![image](https://media.discordapp.net/attachments/1202265337903448174/1212125656549302333/image.png?ex=65f0b2fc&is=65de3dfc&hm=8abb8cd2b705fd345fdf3e4503e2553774fc7b537f4f4ac17348345c37729056&=&format=webp&quality=lossless)

Composer
![image](https://media.discordapp.net/attachments/1202265337903448174/1212122499819507873/image.png?ex=65f0b00c&is=65de3b0c&hm=3699adef5d88b14b168be9e55644237a8879c40768ef909c8de897efcd12b6a9&=&format=webp&quality=lossless)

The error while on the index.php of Phpmyadmin:
![image](https://media.discordapp.net/attachments/1202265337903448174/1212125283012972584/image.png?ex=65f0b2a3&is=65de3da3&hm=429164506d1b7a69c53d3809b5f3469bb3f7d89440a67778dbfa8f0be93b33cf&=&format=webp&quality=lossless)
![image](https://media.discordapp.net/attachments/1202265337903448174/1212125904147447919/image.png?ex=65f0b337&is=65de3e37&hm=c5dde7765d2c2d4c3415fe560a3cf2948d379773ac31e9418da26c48668b3c12&=&format=webp&quality=lossless)

Basically some mess with yarn build..

**Comment by kamil-tekiela** [comment]
On a side note, we should probably consider migrating to a newer Yarn https://yarnpkg.com/migration/guide @MauricioFauth

**Comment by Z3NTL3** [comment]
I wanna set it up for my school project, pls some help would appreciate.

**Comment by ibennetch** [comment]
I really recommend downloading the packaged release files directly from https://www.phpmyadmin.net/downloads/ unless you have a compelling reason not to.

In particular, I really dislike distributing phpMyAdmin through Composer and the build process is somewhat fragile.

**Comment by Z3NTL3** [comment]
tried, but procudes the exact same issue

> I really recommend downloading the packaged release files directly from https://www.phpmyadmin.net/downloads/ unless you have a compelling reason not to.
> 
> In particular, I really dislike distributing phpMyAdmin through Composer and the build process is somewhat fragile.

tried, but procudes the exact same issue

**Comment by liviuconcioiu** [comment]
> tried, but procudes the exact same issue

Can you try and inspect in network monitor each resource it opens? Probably the server is not configured correctly or probably you have an addon that blocks javascript?

**Comment by liviuconcioiu** [comment]
Also, I see in the image you have `unix//run/php/php-fpm.sock`. It should be `unix:/run/php/php-fpm.sock`

**Comment by williamdes** [comment]
What I see is that all files go into the cgi handler. For me this is wrong. You should only send `.php` files.

See https://caddy.community/t/phpmyadmin-is-not-working/12003/4

You are probably missing `file_server`
